network/nodeTracker.py

```diff
@@ -0,0 +1,105 @@
+import argparse
+import json
+import os
+from datetime import datetime
+
+from zenlog import log
+
+
+class NodeTracker:
+	"""
+	Class for tracking the number of nodes in a network.
+	"""
+
+	@staticmethod
+	def _load_time_series_data(output):
+		"""
+		Loads the time series data from the output file.
+		"""
+
+		time_series_data = []
+		if os.path.exists(output) and os.path.getsize(output) > 0:
+			try:
+				with open(output, 'r', encoding='utf8') as infile:
+					time_series_data = json.load(infile)
+			except Exception as ex:
+				raise ValueError(f'Error reading input file: {str(ex)}') from ex
+
+		return time_series_data
+
+	def _add_data_point(self, data_point, output_file):  # pylint: disable=no-self-use
```

Comment:
staticmethod here also

Reply:
I got the lint error: 
`W0211: Static method with 'self' as first argument (bad-staticmethod-argument)`

Actually this method is used "self". I will remove the `# pylint: disable=no-self-use`